consumergroup/offset_manager.go

```diff
@@ -201,7 +203,13 @@ func (zom *zookeeperOffsetManager) offsetCommitter() {
 			close(zom.closed)
 			return
 		case <-tickerChan:
-			zom.commitOffsets()
+			if err := zom.commitOffsets(); err != nil {
+				zom.cg.errors <- &sarama.ConsumerError{
+					Topic:     "",
+					Partition: -1,
+					Err:       err,
+				}
+			}
```

Comment:
Should we wrap it in a ConsumerError? Or change the channel type to just be error?